src/core/draw_tools/atoms/combinedAtom.ts

```diff
@@ -41,21 +41,19 @@ export const combinedAtom = createAtom(
     addFeature: (feature: Feature) => feature,
 
     temporaryGeometryAtom,
-    updateTempFeatures: (features: Feature[], updateIndexes: number[]) => {
-      return { features, indexes: updateIndexes };
-    },
+    updateTempFeatures: (d: { features: Feature[]; indexes: number[] }) => d,
```

Comment:
What does "d" stand for? (Asking because I really don't know)

Reply:
just generic name for pass-through argument, anyway whole thing is planned for rewrite

Reply:
Imo such name can cause confusion if somebody else works on this code. Could at least be obj or val or something like that :)